### ISSUE TITLE
autostop not pausing slides

### ISSUE THREAD

**Issue by steveb533** [andygott/Really-Simple-Slideshow#15]
When you have autostop: true for the next/prev buttons  have the play pause enabled, it switches the button from "pause" to "play" but the slides still auto advance...

If you look at the html-captions-demo.html demo included in the zip and add the autostop: true to the controls you will see what I mean. Thanks,

Steve

**Comment by mateomtb** [comment]
I'm having the same issue.

**Comment by mateomtb** [comment]
To clarify in my use case, if the gallery is in paused state and you click on an index number the gallery begins to auto play again but the play/pause button still shows "play". I'm trying to figure out how to keep everything completely manual, only advancing by the index buttons with no play/pause.

**Comment by mateomtb** [comment]
Sorry figured it out. Set the autostart to false.

**Comment by steveb533** [comment]
Sorry where did you put autostart false? under playPause? I still think this is a bug though..

**Comment by mateomtb** [comment]
I'm not using the auto play feature at all so this may not solve your issue, but this is what I did.

jQuery(document).ready(function () {

```
var opts = {
    controls: {
        //playPause: {auto: false},
        //previousSlide: {auto: true},
        //nextSlide: {auto: true},
        index: {auto: true}

    },

    autostart: false,

    slide_data_selectors: {
        caption: {selector: 'div.caption', attr: false}
    }
};
jQuery('.rs-slideshow').rsfSlideshow(opts);
```

});

**Comment by samford** [comment]
I'm having this issue as well—as long as the slideshow is playing before the previous/next/index links are clicked, the slideshow is only superficially paused (that is, the play/pause text changes to a paused state, but the slideshow continues to play).

As @mateomtb demonstrated, currently the only way to entirely avoid this issue is to set `autostart: false` and disable the play/pause control (`playPause: {auto: false}`)—if the play/pause control is left intact then you will still run into this issue if you play the slideshow and do not pause it before pressing the previous/next links.

As usual, thanks for the great plugin, but this is definitely still an issue, so here's one more request for this to be resolved.

**Comment by andygott** [comment]
Sorry about the late response folks! Been crazy busy around here.

Just a quick note to let you know that I’ll be looking into this at some point in the next couple of days.

Andy

**Comment by alypax** [comment]
Did anyone ever figure this out? I'm having the same issue

**Comment by andygott** [comment]
Hi all. This should be fixed in the latest version.

Closed with 0a9cb91b3b0d172427a7bc8d4d5fd92608f79a6c

**Comment by alypax** [comment]
ahh that did it! Thanks so much!
